src/styles/blocks/calendar.scss

```
@@ -0,0 +1,34 @@
+@import "../utils/mixins.scss";
+
+@import "../utils/variables.scss";
+
+.calendar {
+  max-width: $day__size * 7 + $flex__gap * 6;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: $flex__gap;
+
+  &--position {
+    margin: 0 auto;
+    padding: 20px 0;
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    @include square($day__size);
+
+    border: 1px solid #000;
```

create vars for colors
```suggestion
    border: 1px solid $c-black;
```

---

I used variables for colors which repeats 2 times or more. black color doesnt repeat

---

You have to get used to creating a variable for all colors, bc in production you always do this)